scripts/relup-base-packages.sh

```diff
@@ -0,0 +1,62 @@
+#!/usr/bin/env bash
+if [[ -n "$DEBUG" ]]; then
+    set -x
+fi
+set -euo pipefail
+
+PROFILE="${1}"
+if [ "$PROFILE" = "" ]; then
+    $PROFILE="emqx"
+fi
+
+case $PROFILE in
+    "emqx")
+        BROKER="broker"
```

Comment:
```suggestion
        S3_DIR="broker"
```

Reply:
This is not the path to s3, this is the path to emqx.com, so I've changed it to "DIR"